tests/pgsql/pgsql-7000-ips/test.yaml

```diff
@@ -0,0 +1,533 @@
+      pcap_cnt: 61
+      pgsql.request.simple_query: BEGIN;
+      pgsql.tx_id: 19
+      stream: 1
+      tx_id: 18
+- filter:
+    count: 1
+    match:
+      alert.signature: Pgsql table delete
+      alert.signature_id: 2
+      app_proto: pgsql
+      direction: to_server
+      event_type: alert
+      payload_printable: ......./...O....user.ju-Test.database.Test.application_name.psql.client_encoding.UTF8..p...6SCRAM-SHA-256.... n,,n=,r=ROtF8e2Fme8+eORLNHTwkZaKp...lc=biws,r=ROtF8e2Fme8+eORLNHTwkZaKtpbEaXYJOnd3qt6QNCsAv0wj,p=I4V0zdtQqrxum6B+QzprHHC0nBD+mVtBWpc+arfXa+c=Q....BEGIN;.Q....DELETE FROM new_table WHERE NAME='Remus';.Q...1DELETE FROM new_table WHERE NAME='Londubat';.Q...1DELETE FROM new_table WHERE NAME='Hermione';.Q.../DELETE FROM new_table WHERE NAME='Maugre';.Q....COMMIT;.Q....BEGIN;.Q...HINSERT INTO new_table VALUES('Hermione', 'prof_gramger@gmail.com');.Q...CINSERT INTO new_table VALUES('Remus', 'prof_lupin@gmail.com');.Q....SELECT 1/0;.Q...EINSERT INTO new_table VALUES('Maugre', 'prof_folloy@gmail.com');.Q...IINSERT INTO new_table VALUES('Londubat', 'prof_londubat@gmail.com');.Q....SELECT * FROM new_table;.Q....COMMIT;.
+      pcap_cnt: 61
+      pgsql.request.simple_query: BEGIN;
+      pgsql.tx_id: 19
+      stream: 1
+      tx_id: 18
+- filter:
+    count: 1
+    match:
+      alert.signature: Pgsql table select
+      alert.signature_id: 1
+      app_proto: pgsql
+      direction: to_server
+      event_type: alert
+      payload_printable: ......./...O....user.ju-Test.database.Test.application_name.psql.client_encoding.UTF8..p...6SCRAM-SHA-256.... n,,n=,r=ROtF8e2Fme8+eORLNHTwkZaKp...lc=biws,r=ROtF8e2Fme8+eORLNHTwkZaKtpbEaXYJOnd3qt6QNCsAv0wj,p=I4V0zdtQqrxum6B+QzprHHC0nBD+mVtBWpc+arfXa+c=Q....BEGIN;.Q....DELETE FROM new_table WHERE NAME='Remus';.Q...1DELETE FROM new_table WHERE NAME='Londubat';.Q...1DELETE FROM new_table WHERE NAME='Hermione';.Q.../DELETE FROM new_table WHERE NAME='Maugre';.Q....COMMIT;.Q....BEGIN;.Q...HINSERT INTO new_table VALUES('Hermione', 'prof_gramger@gmail.com');.Q...CINSERT INTO new_table VALUES('Remus', 'prof_lupin@gmail.com');.Q....SELECT 1/0;.Q...EINSERT INTO new_table VALUES('Maugre', 'prof_folloy@gmail.com');.Q...IINSERT INTO new_table VALUES('Londubat', 'prof_londubat@gmail.com');.Q....SELECT * FROM new_table;.Q....COMMIT;.Q....BEGIN;.
+      pcap_cnt: 64
+      pgsql.request.simple_query: INSERT INTO new_table VALUES('Hermione', 'prof_gramger@gmail.com');
```

Comment:
xD

Reply:
Can you tell I was listening to HP books when I created those pcaps? xD